usr/iscsi_util.c

```diff
@@ -97,18 +97,19 @@ int get_score_path()
 int write_oom_score() 
 {
     int fd;
-	int score_path;
+	int score_path,len;
     char score[12];
 
     score_path = get_score_path();
     if (score_path < 0) {
 		snprintf(score, 12, "%d", score_path);
+		len = strlen(score);
```

Comment:
Same as above. You can just do len = snprintf(.... etc

Ideally, you should also check that the return of snprintf is non-negative, so you can catch errors